core/src/com/ericc/the/game/components/StatisticsComponent.java

```diff
@@ -0,0 +1,19 @@
+package com.ericc.the.game.components;
+
+import com.badlogic.ashley.core.Component;
+
+public class StatisticsComponent implements Component{
```

Comment:
Generalnie to nie wiem czy chcemy mieć jeden komponent ze wszystkimi statystykami. Ale to nie kmina na 5 ranko

Reply:
też jestem za rozdzieleniem na 5 komponentów